.scripts/generate_feature_graphic/generate_feature_graphic.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+
+script_location="$(dirname "$(readlink -f "$0")")"
```

Comment:
Since the script should always run in the root directory of the project anyway (no reason to support anything else really) we could also just save the absolute path to `.scripts/generate_feature_graphic/featureGraphic.svg` in a variable; no need for `$script_location` then; idk.

```sh
feature_graphic="$(readlink -f .scripts/generate_feature_graphic/featureGraphic.svg)"
for lang in fastlane/metadata/android/*; do
  ...
  cp "$feature_graphic" ...
  ...
done
```

Reply:
I don't think I agree with this. Sure, this script would normally only be run from CI from one set directory but why remove code that makes the script less reliable and flexible? I'll just leave this code as-is.